src/FormioBaseComponent.ts

```diff
@@ -62,6 +63,7 @@ export class FormioBaseComponent implements OnInit, OnChanges, OnDestroy {
   private submitting = false;
 
   constructor(
+    public customTags: CustomTagsService,
```

Comment:
Can we make this the last component and also optional so we can preserve reverse compatibility?  Example:  The Angular Material library would break with this change @ https://github.com/formio/angular-material-formio/blob/master/projects/angular-material-formio/src/lib/formio.component.ts#L34